<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSetPreOrderDlg</class>
 <widget class="QDialog" name="frmSetPreOrderDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>frmSetPreOrderDlg</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>561</width>
     <height>321</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>预埋（本地，手动发出）</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>预埋（本地，当进入交易状态时发出）</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>条件（本地，当行情满足以下条件时发出）</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_ok">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
color:white;
font: 12pt &quot;宋体&quot;;
        border-radius: 3px;
        border: none;
		background: #2a3a57;
        height:25px;
        width:35px;
        margin-top:0px;
        margin-bottom:5px;
}

/*划过按钮*/
QPushButton:enabled:hover{
        background: #227dc3;
}

QPushButton:enabled:pressed{
        background: #227dc3;
}</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>181</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>9999.989999999999782</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>180</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>时</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>180</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
color:white;
font: 12pt &quot;宋体&quot;;
        border-radius: 3px;
        border: none;
		background: #2a3a57;
        height:25px;
        width:35px;
        margin-top:0px;
        margin-bottom:5px;
}

/*划过按钮*/
QPushButton:enabled:hover{
        background: #227dc3;
}

QPushButton:enabled:pressed{
        background: #227dc3;
}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tip">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>提示</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_3</sender>
   <signal>clicked()</signal>
   <receiver>frmSetPreOrderDlg</receiver>
   <slot>slotReLayout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>frmSetPreOrderDlg</receiver>
   <slot>slotRadio0()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>frmSetPreOrderDlg</receiver>
   <slot>slotRadio1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>frmSetPreOrderDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>frmSetPreOrderDlg</receiver>
   <slot>slotOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotReLayout()</slot>
  <slot>slotRadio0()</slot>
  <slot>slotRadio1()</slot>
  <slot>slotOk()</slot>
 </slots>
</ui>
